webapp/components/sidebar.jsx

```diff
@@ -299,6 +305,9 @@ export default class Sidebar extends React.Component {
             ChannelActions.goToChannel(nextChannel);
             this.updateScrollbarOnChannelChange(nextChannel);
             this.isSwitchingChannel = false;
+        } else if (Utils.cmdOrCtrlPressed(e) && e.shiftKey && e.keyCode === Constants.KeyCodes.K) {
+            e.preventDefault();
```

Comment:
No need to do `e.preventDefault();` here if you're doing it again in `handleOpenMoreDirectChannelsModal ()`